preprocessy/resampling/_split.py

```diff
@@ -21,6 +21,9 @@ class Split:
     def __init__(self):
         pass
 
+    def __repr__(self):
+        pass
```

Comment:
You can use `self.train_size`, `self.test_size` and `self.random_state` along with `None` for the dataframes.